_beats/libbeat/docs/version.asciidoc

```diff
@@ -6,3 +6,4 @@
 :docker: 1.12
 :docker-compose: 1.11
 :branch: 7.0
+:major-version: 7.x
```

Comment:
do you know if we need to do something with this?

Reply:
I don't this so, we don't use this reference as far as I know.